.travis.yml

```diff
@@ -53,6 +53,7 @@ before_install:
   - sudo su $GALAXY_TRAVIS_USER -c 'tar xfz master.tar.gz'
 
 install:
+  - ansible-galaxy install -r requirements_roles.yml -p roles
```

Comment:
@afgane this would need to move before we start building the docker container on line 36.
If you put this at line 31 it should work.